### PR TITLE
Admin: home page: fix typo #7257

### LINKED ISSUE #7257: Admin: home page: fix typo
Change from `Add Instructor` to `Add Instructors` is required as the details are expected from multiple Instructors.
screen shot 2017-05-09 at 11 43 15 am

### PR DESCRIPTION
**Pull request by angelosdmg**
Fixes #7257 

Fixed minor Typo

### CODE REVIEW

**Comment by whipermr5** [comment]
Hi @angelosdmg, thanks for your contribution! You'll have to update all the affected test HTML files using [GodMode](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md) for the CI to pass.

Follow the documentation to configure and enable GodMode, then run the failing tests shown in the Travis CI build log linked below.

**Comment by angelosdmg** [comment]
@whipermr5 Should I open a new PR?

**Comment by whipermr5** [comment]
@angelosdmg No need! Just make the changes, commit and push to the same branch; the new commits will show up in this PR automatically :)

**Comment by whipermr5** [comment]
From the [docs](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md#best-practices):

Please remember to disable GodMode once the necessary changes have been made, before committing the changes.

It should only be enabled locally. After running the required tests to update the HTML files, commit those HTML files, then disable GodMode.

File: src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
 public abstract class BaseUiTestCase extends BaseTestCaseWithBackDoorApiAccess {
 
     /** Indicates if the test-run is to use GodMode. */
-    private static boolean isGodModeEnabled;
+    private static boolean isGodModeEnabled = true;
```
Comment (VamsiSangam) [inline src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java]: God mode is meant to be run only on your local PC. So going by [godmode.md](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md), what you have to do is -

- set `isGodModeEnabled = true;`
- run all CI tests on your PC.
- change `isGodModeEnabled = true;` to `isGodModeEnabled;`
- (Recommended step) Run CI tests again and ensure all tests pass.
- Now add, commit and push your files.

Comment (whipermr5) [inline src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java]: > run all CI tests on your PC.

Actually, only the failed tests in the CI log need to be run.

Comment (angelosdmg) [inline src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java]: still working on it. i'll get you informed asap

Comment (angelosdmg) [inline src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java]: @whipermr5 
@VamsiSangam 
I get 5 errors in Ci Tests. Should I post it here or open a new issue?

Comment (VamsiSangam) [inline src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java]: Make a comment here :+1:

Comment (wkurniawan07) [inline src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java]: @angelosdmg you need to fix it here, as such errors are likely to be caused by the changes you introduced. If you believe there are unrelated errors, feel free to post it here.

**Comment by chowyb** [comment]
@angelosdmg it seems like GitHub no longer recognizes the branch which you plan to merge from. May I check if you are still working on this PR?

**Comment by angelosdmg** [comment]
@chowyb I'm still working and I'll fix this. Maybe I made a mistake and deleted. Sorry for the inconvenience. Fixing it ASAP

**Comment by angelosdmg** [comment]
-Part of the output I get when running the CiTests in the shell

> 
614 tests completed, 6 failed, 87 skipped
There were failing tests
:killFirefox1
:testTry2

teammates.test.cases.browsertests.InstructorCourseJoinConfirmationPageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at InstructorCourseJoinConfirmationPageUiTest.java:32

teammates.test.cases.browsertests.StudentProfilePageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at StudentProfilePageUiTest.java:43

teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at InstructorFeedbackEditPageUiTest.java:46

teammates.test.cases.browsertests.SystemErrorEmailReportTest.classSetup FAILED
    org.openqa.selenium.TimeoutException at SystemErrorEmailReportTest.java:29

teammates.test.cases.browsertests.AllJsTests.classSetup FAILED
    org.openqa.selenium.TimeoutException at AllJsTests.java:30

teammates.test.cases.testdriver.BackDoorTest.classSetup FAILED
    java.lang.AssertionError at BackDoorTest.java:32

93 tests completed, 6 failed, 87 skipped
There were failing tests
:killFirefox2
:testTry3

teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at InstructorFeedbackEditPageUiTest.java:46

teammates.test.cases.browsertests.InstructorCourseJoinConfirmationPageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at InstructorCourseJoinConfirmationPageUiTest.java:32

teammates.test.cases.browsertests.StudentProfilePageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at StudentProfilePageUiTest.java:43

teammates.test.cases.testdriver.BackDoorTest.classSetup FAILED
    java.lang.AssertionError at BackDoorTest.java:32

teammates.test.cases.browsertests.SystemErrorEmailReportTest.classSetup FAILED
    org.openqa.selenium.TimeoutException at SystemErrorEmailReportTest.java:29

teammates.test.cases.browsertests.AllJsTests.classSetup FAILED
    org.openqa.selenium.TimeoutException at AllJsTests.java:30

93 tests completed, 6 failed, 87 skipped
There were failing tests
:killFirefox3
:testTry4

java.lang.AssertionError: expected:<[BACKDOOR_STATUS_SUCCESS]> but was:<
java.net.ConnectException: Connection refused: connect
        at java.net.DualStackPlainSocketImpl.connect0(Native Method)
        at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
        at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:579)
        at java.net.Socket.connect(Socket.java:528)
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
        at sun.net.www.http.HttpClient.New(HttpClient.java:308)
        at sun.net.www.http.HttpClient.New(HttpClient.java:326)
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:997)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:933)
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:851)
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1092)
        at teammates.test.driver.BackDoor.sendRequest(BackDoor.java:443)
        at teammates.test.driver.BackDoor.makePostRequest(BackDoor.java:422)
        at teammates.test.driver.BackDoor.removeAndRestoreDataBundle(BackDoor.java:76)
        at teammates.test.cases.BaseTestCaseWithBackDoorApiAccess.doRemoveAndRestoreDataBundle(BaseTestCaseWithBackDoorApiAccess.java:67)
        at teammates.test.cases.BaseTestCaseWithDatastoreAccess.removeAndRestoreDataBundle(BaseTestCaseWithDatastoreAccess.java:258)
        at teammates.test.cases.browsertests.InstructorCourseJoinConfirmationPageUiTest.prepareTestData(InstructorCourseJoinConfirmationPageUiTest.java:32)
        at teammates.test.cases.browsertests.BaseUiTestCase.baseClassSetup(BaseUiTestCase.java:49)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
        at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:517)
        at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
        at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
        at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
        at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
        at org.testng.TestRunner.privateRun(TestRunner.java:771)
        at org.testng.TestRunner.run(TestRunner.java:621)
        at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
        at org.testng.SuiteRunner.access$000(SuiteRunner.java:38)
        at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
        at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
>
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
        at teammates.test.cases.BaseTestCase.assertEquals(BaseTestCase.java:133)
        at teammates.test.cases.BaseTestCaseWithDatastoreAccess.removeAndRestoreDataBundle(BaseTestCaseWithDatastoreAccess.java:260)
        at teammates.test.cases.browsertests.InstructorCourseJoinConfirmationPageUiTest.prepareTestData(InstructorCourseJoinConfirmationPageUiTest.java:32)
        at teammates.test.cases.browsertests.BaseUiTestCase.baseClassSetup(BaseUiTestCase.java:49)
        ...

teammates.test.cases.browsertests.InstructorCourseJoinConfirmationPageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at InstructorCourseJoinConfirmationPageUiTest.java:32

java.lang.AssertionError: expected:<[BACKDOOR_STATUS_SUCCESS]> but was:<
java.net.ConnectException: Connection refused: connect
        at java.net.DualStackPlainSocketImpl.connect0(Native Method)
        at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
        at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:579)
        at java.net.Socket.connect(Socket.java:528)
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
        at sun.net.www.http.HttpClient.New(HttpClient.java:308)
        at sun.net.www.http.HttpClient.New(HttpClient.java:326)
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:997)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:933)
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:851)
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1092)
        at teammates.test.driver.BackDoor.sendRequest(BackDoor.java:443)
        at teammates.test.driver.BackDoor.makePostRequest(BackDoor.java:422)
        at teammates.test.driver.BackDoor.removeAndRestoreDataBundle(BackDoor.java:76)
        at teammates.test.cases.BaseTestCaseWithBackDoorApiAccess.doRemoveAndRestoreDataBundle(BaseTestCaseWithBackDoorApiAccess.java:67)
        at teammates.test.cases.BaseTestCaseWithDatastoreAccess.removeAndRestoreDataBundle(BaseTestCaseWithDatastoreAccess.java:258)
        at teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest.prepareTestData(InstructorFeedbackEditPageUiTest.java:46)
        at teammates.test.cases.browsertests.BaseUiTestCase.baseClassSetup(BaseUiTestCase.java:49)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
        at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:517)
        at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
        at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
        at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
        at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
>
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
        at teammates.test.cases.BaseTestCase.assertEquals(BaseTestCase.java:133)
        at teammates.test.cases.BaseTestCaseWithDatastoreAccess.removeAndRestoreDataBundle(BaseTestCaseWithDatastoreAccess.java:260)
        at teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest.prepareTestData(InstructorFeedbackEditPageUiTest.java:46)
        at teammates.test.cases.browsertests.BaseUiTestCase.baseClassSetup(BaseUiTestCase.java:49)
        ...

teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at InstructorFeedbackEditPageUiTest.java:46

java.lang.AssertionError: expected:<[BACKDOOR_STATUS_SUCCESS]> but was:<
java.net.ConnectException: Connection refused: connect
        at java.net.DualStackPlainSocketImpl.connect0(Native Method)
        at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
        at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:579)
        at java.net.Socket.connect(Socket.java:528)
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
        at sun.net.www.http.HttpClient.New(HttpClient.java:308)
        at sun.net.www.http.HttpClient.New(HttpClient.java:326)
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:997)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:933)
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:851)
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1092)
        at teammates.test.driver.BackDoor.sendRequest(BackDoor.java:443)
        at teammates.test.driver.BackDoor.makePostRequest(BackDoor.java:422)
        at teammates.test.driver.BackDoor.removeAndRestoreDataBundle(BackDoor.java:76)
        at teammates.test.cases.BaseTestCaseWithBackDoorApiAccess.doRemoveAndRestoreDataBundle(BaseTestCaseWithBackDoorApiAccess.java:67)
        at teammates.test.cases.BaseTestCaseWithDatastoreAccess.removeAndRestoreDataBundle(BaseTestCaseWithDatastoreAccess.java:258)
        at teammates.test.cases.browsertests.StudentProfilePageUiTest.prepareTestData(StudentProfilePageUiTest.java:43)
        at teammates.test.cases.browsertests.BaseUiTestCase.baseClassSetup(BaseUiTestCase.java:49)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
        at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:517)
        at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
        at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
        at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
        at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
>
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
        at teammates.test.cases.BaseTestCase.assertEquals(BaseTestCase.java:133)
        at teammates.test.cases.BaseTestCaseWithDatastoreAccess.removeAndRestoreDataBundle(BaseTestCaseWithDatastoreAccess.java:260)
        at teammates.test.cases.browsertests.StudentProfilePageUiTest.prepareTestData(StudentProfilePageUiTest.java:43)
        at teammates.test.cases.browsertests.BaseUiTestCase.baseClassSetup(BaseUiTestCase.java:49)
        ...

teammates.test.cases.browsertests.StudentProfilePageUiTest.baseClassSetup FAILED
    java.lang.AssertionError at StudentProfilePageUiTest.java:43

java.lang.AssertionError: expected:<[BACKDOOR_STATUS_SUCCESS]> but was:<
java.net.ConnectException: Connection refused: connect
        at java.net.DualStackPlainSocketImpl.connect0(Native Method)
        at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
        at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:579)
        at java.net.Socket.connect(Socket.java:528)
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
        at sun.net.www.http.HttpClient.New(HttpClient.java:308)
        at sun.net.www.http.HttpClient.New(HttpClient.java:326)
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:997)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:933)
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:851)
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1092)
        at teammates.test.driver.BackDoor.sendRequest(BackDoor.java:443)
        at teammates.test.driver.BackDoor.makePostRequest(BackDoor.java:422)
        at teammates.test.driver.BackDoor.removeAndRestoreDataBundle(BackDoor.java:76)
        at teammates.test.cases.BaseTestCaseWithBackDoorApiAccess.doRemoveAndRestoreDataBundle(BaseTestCaseWithBackDoorApiAccess.java:67)
        at teammates.test.cases.BaseTestCaseWithDatastoreAccess.removeAndRestoreDataBundle(BaseTestCaseWithDatastoreAccess.java:258)
        at teammates.test.cases.testdriver.BackDoorTest.classSetup(BackDoorTest.java:32)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
        at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:517)
        at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
        at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
        at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
        at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
>
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
        at teammates.test.cases.BaseTestCase.assertEquals(BaseTestCase.java:133)
        at teammates.test.cases.BaseTestCaseWithDatastoreAccess.removeAndRestoreDataBundle(BaseTestCaseWithDatastoreAccess.java:260)
        at teammates.test.cases.testdriver.BackDoorTest.classSetup(BackDoorTest.java:32)
        ...

teammates.test.cases.testdriver.BackDoorTest.classSetup FAILED
    java.lang.AssertionError at BackDoorTest.java:32

org.gradle.messaging.remote.internal.PlaceholderException: Timed out after 15 seconds waiting for teammates.test.pageobjects.AppPage$1@6e3d5d7
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'DESKTOP-KHC5TQV', ip: '192.168.1.101', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.7.0_80'
Session ID: 79f1e8f0-c306-44ab-b4b0-2676d57d6834
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=WINDOWS, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=46.0}]
        at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
        at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:261)
        at teammates.test.pageobjects.AppPage.waitForPageToLoad(AppPage.java:180)
        at teammates.test.cases.browsertests.BaseUiTestCase.logout(BaseUiTestCase.java:134)
        at teammates.test.cases.browsertests.BaseUiTestCase.loginAdminToPage(BaseUiTestCase.java:100)
        at teammates.test.cases.browsertests.BaseUiTestCase.loginAdmin(BaseUiTestCase.java:139)
        at teammates.test.cases.browsertests.SystemErrorEmailReportTest.classSetup(SystemErrorEmailReportTest.java:29)
        ...

teammates.test.cases.browsertests.SystemErrorEmailReportTest.classSetup FAILED
    org.openqa.selenium.TimeoutException at SystemErrorEmailReportTest.java:29

org.gradle.messaging.remote.internal.PlaceholderException: Timed out after 15 seconds waiting for teammates.test.pageobjects.AppPage$1@c952270
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'DESKTOP-KHC5TQV', ip: '192.168.1.101', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.7.0_80'
Session ID: 71b538fa-6358-4612-b4db-0a8e9cf35ef7
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=WINDOWS, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=46.0}]
        at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
        at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:261)
        at teammates.test.pageobjects.AppPage.waitForPageToLoad(AppPage.java:180)
        at teammates.test.cases.browsertests.BaseUiTestCase.logout(BaseUiTestCase.java:134)
        at teammates.test.cases.browsertests.BaseUiTestCase.loginAdminToPage(BaseUiTestCase.java:100)
        at teammates.test.cases.browsertests.BaseUiTestCase.loginAdmin(BaseUiTestCase.java:139)
        at teammates.test.cases.browsertests.AllJsTests.classSetup(AllJsTests.java:30)
        ...

teammates.test.cases.browsertests.AllJsTests.classSetup FAILED
    org.openqa.selenium.TimeoutException at AllJsTests.java:30

93 tests completed, 6 failed, 87 skipped
:testTry4 FAILED
:killFirefox4

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':testTry4'.
> There were failing tests

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 10 mins 49.187 secs
~\Desktop\teammates\teammates_original\teammates [7257-fix-typo]>

**Comment by whipermr5** [comment]
@angelosdmg You need to start the dev server before running the tests. Also, you don't actually need to run the entire test suite - the first Travis log shows just one failing test, and that is the only test you need to run.

**Comment by angelosdmg** [comment]
errors

When trying to add an instructor, the ajax request always fails!

**Comment by whipermr5** [comment]
@angelosdmg Have you built the JavaScript files with `npm run build`?

**Comment by angelosdmg** [comment]
I'm encountering some problems: https://github.com/TEAMMATES/teammates/issues/7437
If you think that my pull request is a problem and you want to close it, it's ok but please help me if you don't mind. 
Thank you

**Comment by whipermr5** [comment]
@angelosdmg no worries; fix your setup first then continue this PR :)

**Comment by whipermr5** [comment]
@angelosdmg Any updates on this?

**Comment by angelosdmg** [comment]
There doesn't seem to be a fix in my problem and I can't spend time on this right now. Please gice this chance to someone else and I will come back with another firsttimer. Sorry for the confusion

**Pull request closed without merging**